src/converters/jpeg_to_bmp.c

```diff
@@ -0,0 +1,119 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <jpeglib.h>
+
+int jpeg_to_bmp(const char *input_filename, const char *output_filename)
+{
+    FILE *input_file = fopen(input_filename, "rb");
+    if (!input_file)
+    {
+        fprintf(stderr, "Error: could not open input file %s\n", input_filename);
+        return 1;
+    }
+    
+    // Initialize JPEG decompressor object
```

Comment:
change capitalisation on comments to be inline with other files
// Initialize JPEG decompressor object -> // initlialize JPEG decompressor object